### DISCUSSION TITLE
Why are some features only in master and not in the stable releases?

### DISCUSSION THREAD

**Discussion by davidmurdoch** [jellyfin/jellyfin-web#4128] [Q&A] [answered]
Specifically, i'd love for https://github.com/jellyfin/jellyfin-web/commit/fe05939ec3b46011be1d074456f119e1cf0201b1 to land in 10.8

**Comment by Shadowghost** [comment, marked as answer]
Because master is the development branch. When it's stable enough for a release, the release branch gets forked from it and includes all changes since the last stable release.
Once a release branch was created it only gets bugfixes applied to it, all new features go into the master branch again and the cycle continues.

**Reply by davidmurdoch** [reply]
Ah, so because fe05939 is more of a "feature" it won't land until v10.9. Cool, and thanks!
